tests/testutil/service.go

```diff
@@ -202,6 +203,98 @@ func (ts *TestService) GetRunningPods() string {
 	return out
 }
 
+// GetEndpointURL returns the URL for this service's app if the App object is versioned it returns
+// the versioned url.
```

Comment:
This is incorrect, it only returns the service url. Service == versioned, app == unversioned. Let's not use the 'versioned' or 'unversioned' phrases, that can get confusing, let's just use 'service' or 'app'.